src/AddTodoForm.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function AddTodoForm() {
+    return (
+        <form>
+            <label htmlFor="todoTitle">Title: </label>
+            <input type="text" id="todoTitle"></input>
+            <input type="submit" value="Add"></input>
```

Comment:
An input HTML element has a self-contained syntax, meaning that it doesn't contain any content and does not require a separate closing tag. For example:
`<input type="text" />`

Reply:
Hi Gonzalo, I have made the changes. Thank you @gsvidal